test/new-e2e/tests/agent-subcommands/flare/flare_nix_test.go

```diff
@@ -27,24 +26,22 @@ var systemProbeConfiguration []byte
 //go:embed fixtures/security-agent.yaml
 var securityAgentConfiguration []byte
 
-type linuxFlareSuite struct {
-	baseFlareSuite
-}
+type linuxFlareSuite baseFlareSuite
 
 func TestLinuxFlareSuite(t *testing.T) {
-	e2e.Run(t, &linuxFlareSuite{}, e2e.FakeIntakeStackDef(e2e.WithVMParams(ec2params.WithOS(ec2os.UbuntuOS))))
+	t.Parallel()
```

Comment:
I think we already run tests in different jobs, I would rather avoid this unexpected change

Reply:
Merge conflict, thanks you for spotting.
`t.Parallel()` used to be there: https://github.com/DataDog/datadog-agent/commit/b6901b6bc5e403035b9b411dc9508d992a0c4d9c#diff-752a805b7748973073858f2aec7ffbe8dc468162f62357acbe485d9a8b5ef860L36